pandas/core/ops/invalid.py

```diff
@@ -7,17 +7,26 @@
 import operator
 from typing import (
     TYPE_CHECKING,
+    Any,
     Callable,
     NoReturn,
 )
 
 import numpy as np
 
 if TYPE_CHECKING:
-    from pandas._typing import npt
+    from pandas._typing import (
+        ArrayLike,
+        Scalar,
+        npt,
+    )
 
 
-def invalid_comparison(left, right, op) -> npt.NDArray[np.bool_]:
+def invalid_comparison(
+    left: ArrayLike,
+    right: ArrayLike | Scalar,
+    op: Callable[[Any, Any], bool],  # Can we reuse _BoolOp here?
```

Comment:
What do you mean with this comment?

Typeshed does not use such an alias: https://github.com/python/typeshed/blob/4c5c5760cfa7c9707383b2e6fe4faf021911e08c/stdlib/_operator.pyi#L48

Reply:
I noticed the same thing as well. `_BoolOp` is defined and used locally [here](https://github.com/pandas-dev/pandas/blob/main/pandas/_libs/ops.pyi#L20), so I would like to ask if we can somehow reuse this type or it's fine with `Callable[[Any, Any], bool]`